### PR TITLE
[ar] Unsupervised Learning

### PR DESCRIPTION
**Pull request by redouanelg**
Unsupervised learning, in progress

### CODE REVIEW

**Comment by redouanelg** [comment]
Hi @shervinea, done with the translation. Thank you.

**Comment by shervinea** [comment]
Thanks for your work @redouanelg! Now, let's start the review process.

**Comment by qunaieer** [comment]
Hello @shervinea 
How can I review this file? What is the process? I am not sure from where should I download it.

**Comment by shervinea** [comment]
Hi @qunaieer, thank you for helping out with this review! 

You can go to the ['Files' tab](https://github.com/shervinea/cheatsheet-translation/pull/88/files) and write down comments when applicable using the '+' button of the associated lines.

File: ar/cheatsheet-unsupervised-learning.md [hunk 1]
```diff
@@ -1,340 +1,401 @@
...
+</div>
 <br>
 
 **51. The Machine Learning cheatsheets are now available in Arabic.**
 
-&#10230;
-
+<div dir="rtl">
+ورقات المراجعة للتعلم الآلي متوفرة حاليا باللغة العربية
+</div>
```
Comment (qunaieer) [inline ar/cheatsheet-unsupervised-learning.md]: المرجع السريع لتعلم الآلة متوفر الآن باللغة العربية.

File: ar/cheatsheet-unsupervised-learning.md [hunk 2]
```diff
@@ -1,340 +1,401 @@
...
 <br>
 
 **53. Translated by X, Y and Z**
 
-&#10230;
-
+<div dir="rtl">
+تم ترجمته بواسطة X, Y و Z </div>
+</div>
```
Comment (qunaieer) [inline ar/cheatsheet-unsupervised-learning.md]: تمت الترجمة بواسطة X,Y و Z

File: ar/cheatsheet-unsupervised-learning.md [hunk 3]
```diff
@@ -1,340 +1,401 @@
...
 <br>
 
 **54. Reviewed by X, Y and Z**
 
-&#10230;
-
+<div dir="rtl">
+تم مراجعته بواسطة X, Y و Z </div>
```
Comment (qunaieer) [inline ar/cheatsheet-unsupervised-learning.md]: تمت المراجعة بواسطة X,Y و Z

File: ar/cheatsheet-unsupervised-learning.md [hunk 4]
```diff
@@ -1,340 +1,401 @@
...
 <br>
 
 **55. [Introduction, Motivation, Jensen's inequality]**
 
-&#10230;
-
+<div dir="rtl">
+[تقديم، تحفيز، متفاوتة جنسن]
+</div>
```
Comment (qunaieer) [inline ar/cheatsheet-unsupervised-learning.md]: [مقدمة، الحافز، متباينة جينسن]

File: ar/cheatsheet-unsupervised-learning.md [hunk 5]
```diff
@@ -1,340 +1,401 @@
...
-&#10230;
-
+<div dir="rtl">
+[تجميع,
+  التوقع-التعظيم
+  , k-متوسطات
+  , التجميع الهرمي
+  , مقاييس]
+  
+</div>
```
Comment (qunaieer) [inline ar/cheatsheet-unsupervised-learning.md]: [التجميع، تعظيم القيمة المتوقعة،  تجميع k-متوسطات، التجميع الهرمي، مقاييس]

File: ar/cheatsheet-unsupervised-learning.md [hunk 6]
```diff
@@ -1,340 +1,401 @@
...
 <br>
 
 **57. [Dimension reduction, PCA, ICA]**
 
-&#10230;
+<div dir="rtl">
+[خفض الأبعاد, PCA, ICA]
+</div>
+<br>
```
Comment (qunaieer) [inline ar/cheatsheet-unsupervised-learning.md]: [تقليص الأبعاد، تحليل المكون الرئيس (PCA)، تحليل المكونات المستقلة (ICA)]

**Comment by redouanelg** [comment]
Thanks a lot dear @qunaieer for your high quality work. 

Dear @shervinea, how should I proceed, can you approve the reviews? or should I do something on my side? Thanks.

**Comment by shervinea** [comment]
Thanks @qunaieer for your thorough work and thanks @redouanelg for reaching out.

@redouanelg each time you agree with a suggestion, you can implement the associated change on the `.md` file and then mark the corresponding conversation as 'resolved'.

**Comment by redouanelg** [comment]
Done, thank you all.

**Pull request merged**